tests/testthat/test-hover_reload_button.R

```diff
@@ -0,0 +1,33 @@
+context("Reload functionality")
```

Comment:
Let's remove the `context` call, the tidyverse team has deprecated this function and actually discourages using it.

* https://testthat.r-lib.org/reference/context.html
* https://www.tidyverse.org/blog/2019/04/testthat-2-1-0/#context-is-now-optional